src/strings/en-us.json

```diff
@@ -789,6 +789,8 @@
     "LabelPublicHttpsPortHelp": "The public port number that should be mapped to the local HTTPS port.",
     "LabelPublishedServerUri": "Published Server URIs",
     "LabelPublishedServerUriHelp": "Override the URI used by Jellyfin, based on the interface, or client IP address.",
+    "LabelQsvDevice": "QSV Device",
+    "LabelQsvDeviceHelp": "This is the render node that is used for hardware acceleration.",
```

Comment:
Please add a slightly more detailed description of what the user should fill in. And emphasize that this should be left blank by default.

Reply:
Hows something like this?

"LabelQsvDeviceHelp": "Specify the device for Intel QSV. On Linux, this is the render node, e.g., /dev/dri/renderD128. On Windows, this is the device index starting from 0. Leave blank to use the default device."

Reply:
`"LabelQsvDeviceHelp": "Specify the device for Intel QSV on a multi-GPU system. On Linux, this is the render node, e.g., /dev/dri/renderD128. On Windows, this is the device index starting from 0. Leave blank unless you know what you are doing."`

Made some changes to keep away newbie users from touching this.